postgres/infoschema.go

```diff
@@ -336,6 +343,83 @@ func getConstraints(conv *internal.Conv, db *sql.DB, table schemaAndName) ([]str
 	return primaryKeys, m, nil
 }
 
+type fkConstraint struct {
+	name    string
+	table   string
+	refcols []string
+	cols    []string
+}
+
+// getForeignKeys return list all the foreign keys constraints.
```

Comment:
Suggest rephrasing:

// getForeignKeys returns a list all the foreign keys constraints.

Reply:
updated comment